.github/workflows/deploy.yml

```diff
@@ -0,0 +1,78 @@
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_DASHBOARD }}
+          docker build -t dashboard:latest ./dashboard
+          docker tag dashboard:latest ${{ secrets.ECR_DASHBOARD }}:latest
+          docker push ${{ secrets.ECR_DASHBOARD }}:latest
+
+      - name: Build and Push Quadhoral-Openmeteo Docker Image
+        if: ${{ hashFiles('quadhoral-openmeteo/Dockerfile') != '' }}
+        run: |
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_QUADHORAL_OPENMETEO }}
+          docker build -t quadhoral-openmeteo:latest ./quadhoral-openmeteo
+          docker tag quadhoral-openmeteo:latest ${{ secrets.ECR_QUADHORAL_OPENMETEO }}:latest
+          docker push ${{ secrets.ECR_QUADHORAL_OPENMETEO }}:latest
+
+      - name: Build and Push Weekly-Openmeteo Docker Image
+        if: ${{ hashFiles('weekly-openmeteo/Dockerfile') != '' }}
+        run: |
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_WEEKLY_OPENMETEO }}
+          docker build -t weekly-openmeteo:latest ./weekly-openmeteo
+          docker tag weekly-openmeteo:latest ${{ secrets.ECR_WEEKLY_OPENMETEO }}:latest
+          docker push ${{ secrets.ECR_WEEKLY_OPENMETEO }}:latest
+
+      - name: Build and Push Astronomy Pipeline Docker Image
+        if: ${{ hashFiles('astronomy-pipeline/Dockerfile') != '' }}
+        run: |
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_ASTRONOMY_PIPELINE }}
+          docker build -t astronomy-pipeline:latest ./astronomy-pipeline
+          docker tag astronomy-pipeline:latest ${{ secrets.ECR_ASTRONOMY_PIPELINE }}:latest
+          docker push ${{ secrets.ECR_ASTRONOMY_PIPELINE }}:latest
+
+      - name: Build and Push Aurorawatch Pipeline Docker Image
+        if: ${{ hashFiles('aurorawatch-pipeline/Dockerfile') != '' }}
+        run: |
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_AURORAWATCH_PIPELINE }}
+          docker build -t aurorawatch-pipeline:latest ./aurorawatch-pipeline
+          docker tag aurorawatch-pipeline:latest ${{ secrets.ECR_AURORAWATCH_PIPELINE }}:latest
+          docker push ${{ secrets.ECR_AURORAWATCH_PIPELINE }}:latest
+
+      - name: Build and Push SMS-Checker Docker Image
+        if: ${{ hashFiles('sms-checker/Dockerfile') != '' }}
+        run: |
+          aws ecr get-login-password --region ${{ secrets.AWS_REGION }} | docker login --username AWS --password-stdin ${{ secrets.ECR_SMS_CHECKER }}
+          docker build -t sms-checker:latest ./sms-checker
+          docker tag sms-checker:latest ${{ secrets.ECR_SMS_CHECKER }}:latest
+          docker push ${{ secrets.ECR_SMS_CHECKER }}:latest
```

Comment:
Accounts for all the images of our future pipelines in their respective ECRs. Will be very useful in saving time acorss the deployment of our many pipelines.